Python/View_and_Combine_STL/README.md

```diff
@@ -0,0 +1,24 @@
+# View and Combine STL
+
+STL files are an important part of 3D asset creation and management. STL files provide us with basic maps of 3D objects created/edited in applications like Blender/Unreal/Unity
+
+STL are mainly used to create 3D maps of environments like cityscapes and heightmaps of terrain. 
+
+## Running the Script
+* Create a virtual env
+* install numpy-stl, numpy and matplotlib or use requirements.txt
+* You can use combine.py to concatenate multiple STL files(recommended for assets designed in same 3D space or domain)
+* Or you can simply run view.py to view the STL file in a pyplot env.(set the target size to your liking)
+
+## Exploring the STL Space
+[City Generator](https://probabletrain.itch.io/city-generator) is a website that uses tensor fields to generate a 3D map, all are created procedurally and this would be a great space if you want to explore STL and basic 3D enviroments.
+
+When you use City Generator, kindly delete or store the domain.stl file before running the combine.py script as it will result in the creation of flat mesh instead of a 3D one.
+
+Further you can use [Blender](https://www.blender.org/) which is one of the most amazing and Open Source 3D asset and animation creation tool out there.
+
+![3D Cityscape in paint3D](https://github.com/HarshCasper/Rotten-Scripts/blob/master/Python/View_and_Combine_STL/images/building_3d.png)
+![STL View in Python](https://github.com/HarshCasper/Rotten-Scripts/blob/master/Python/View_and_Combine_STL/images/building_py_view.png)
+
+![Road 3D](https://github.com/HarshCasper/Rotten-Scripts/blob/master/Python/View_and_Combine_STL/images/road_3d.PNG)
+![Road Py View](https://github.com/HarshCasper/Rotten-Scripts/blob/master/Python/View_and_Combine_STL/images/road_py_view.PNG)
```

Comment:
Please upload all your images on Imgur and reference it here

Reply:
Actually I tried using Imgur before this but for some reason the embedding didn't work and the images were not loading.

Reply:
Can you please have a look at other PRs that have done the same thing. 

Please go to this [link](https://imgur.com/upload), upload the Image and take the Image URL and add it above.

Reply:
Resolved